### PR TITLE
Quadratic deep as sequential

### PR DESCRIPTION
**Pull request by f-dangel**
Reformulate the `quadratic` test problem as empirical risk minimization with a sequential neural network and square loss, see the [issue in my fork](https://github.com/f-dangel/DeepOBS/issues/1#issue-521076390) for more details.

**Important:** This will change the loss function of the test problem by a constant scale.

### CODE REVIEW

**Comment by fsschneider** [comment]
Thanks for implementing this. I'll merge it once I re-write it in the TensorFlow code as well. That way, the test problem is consistent across frameworks again.
